### PR TITLE
Version 9.4.0rc3

### PR DESCRIPTION
**Pull request by beenje**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beenje** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -35,7 +35,8 @@ requirements:
     # boost max_pin set in pin_run_as_build in conda-forge-pinning
     - boost
     - {{ pin_compatible('numpy') }}
-    - six >=1.10
+    - packaging
+    - psutil
```
Comment (ajoubertza) [inline recipe/meta.yaml:39]: Should we remove `psutil` from the test requires?

Comment (ajoubertza) [inline recipe/meta.yaml:39]: Should we remove `psutil` from the test requires?

Comment (ajoubertza) [inline recipe/meta.yaml:39]: Should we remove `psutil` from the test requires?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -35,7 +35,8 @@ requirements:
     # boost max_pin set in pin_run_as_build in conda-forge-pinning
```
Comment (ajoubertza) [inline recipe/meta.yaml:35]: In the host requirements, we should change cpptango >= 9.4.1.  9.4.0 won't work properly.

File: recipe/meta.yaml (hunk 1 above)
Comment (beenje) [reply recipe/meta.yaml:39]: yes, good point.

File: recipe/meta.yaml (hunk 2 above)
Comment (beenje) [reply recipe/meta.yaml:35]: Of course. I forgot :-(

**Review by ajoubertza — APPROVED** [review]
LGTM.  Thanks!

**Pull request merged**